dependencies.gradle

```diff
@@ -65,10 +65,12 @@ def VERSIONS = [
         'org.mockito:mockito-core:latest.release',
         'org.mockito:mockito-inline:latest.release',
         'org.mongodb:mongo-java-driver:latest.release',
+        'org.neo4j.driver:neo4j-java-driver:4.1.+',
```

Comment:
Can we keep this to `latest.release` and allow dependency locking to do it's thing?